src/services/textChanges.ts

```diff
@@ -686,7 +686,11 @@ namespace ts.textChanges {
         }
 
         public insertExportModifier(sourceFile: SourceFile, node: DeclarationStatement | VariableStatement): void {
-            this.insertText(sourceFile, node.getStart(sourceFile), "export ");
+            this.insertExportModifierAt(sourceFile, node.getStart(sourceFile));
+        }
+
+        public insertExportModifierAt(sourceFile: SourceFile, position: number): void {

```

Comment:
I just don't quite know if you need this, but I don't understand the `start + 1` thing yet.

Reply:
I want to add an export identifier in front of non-exported type, you can see the codeFixImportNonExportedMember1.ts, an export added before function bar(), the start there is the position of line break, so need to +1 for start.